components/TemplateCard/TemplateCard.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components"
+import { AiOutlineDelete as Delete } from "react-icons/ai"
+import Link from "next/link"
+import { deleteTemplate } from "@/utils/helpers"
+
+const TemplateContainer = styled.section`
+    width: 100%;
+    display: grid;
+    grid-template-columns: 1fr 0.5fr 0.25fr;
+    align-items: center;
+    padding-left: 0.75rem;
+    margin-bottom: 0.75rem;
+`
+const IconContainer = styled.div`
+font-size: 1.75rem;
```

Comment:
There seems to be inconsistent formatting when comparing this styled component to the one above. I think we want the styles here to be indented like the one above.

Reply:
My Formatter isn't working so I have to always manually run the command for prettier. Forgot it this time obviously. 
Tried many things to resolve this issue but nothing worked. But I'm not complaining.